src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowPingSpec.groovy

```diff
@@ -94,6 +95,50 @@ class FlowPingSpec extends HealthCheckSpecification {
         swPair = new SwitchPair(src: srcSwitch, dst: dstSwitch, paths: [])
     }
 
+    @Tidy
+    @Unroll("Able to ping a flow with vxlan between switches #swPair.toString()")
+    @Tags([TOPOLOGY_DEPENDENT])
+    def "Able to ping a flow with vxlan"(Switch srcSwitch, Switch dstSwitch) {
+        given: "A flow with random vxlan"
+        def flow = flowHelperV2.randomFlow(srcSwitch, dstSwitch)
+        flow.encapsulationType=  FlowEncapsulationType.VXLAN
```

Comment:
I think you need to ensure that switches support VXLAN.  try to use `isVxlanEnabled`

See on this test as on example 
```
    def "Able to synchronize switch with 'vxlan' rule(install missing rules and meters)"() {
        given: "Two active not neighboring Noviflow switches"
        def switchPair = topologyHelper.getAllNotNeighboringSwitchPairs().find { swP ->
            swP.paths.find { path ->
                pathHelper.getInvolvedSwitches(path).every { switchHelper.isVxlanEnabled(it.dpId) }
            }
        } ?: assumeTrue(false, "Unable to find required switches in topology")

```

Reply:
solved